### ISSUE TITLE
I don't get how to switch to "edit" mode

### ISSUE THREAD

**Issue by stevepiron** [devbridge/Styleguide#163]
Hello,

I'm not sure how I should proceed to switch to edit mode. The styleguide server is listening on port 8889 after running `gulp start-styleguide` and I can see the styleguide at `project.dev/styleguide/`.

**Comment by stevepiron** [comment]
Ok, as I ran `styleguide initialize` at the root of my project directory but actually wanted it (the styleguide folder) to be in an existing directory, what I did was move it to that directory. But this did broke some things, it seems. I deleted the styleguide folder and ran `styleguide initialize public/styleguide` to recreate the styleguide folder with correct paths and now when I run `gulp start-styleguide` edit view is the default.

You should warn people about this. I'll leave this issue open so that you're aware of this but feel free to close it.

**Comment by yumyo** [comment]
Yep, paths were confusing for me too. An hour wasted before sorting out it was that simple. 
+ 1 to add it to the docs

**Comment by nynban** [comment]
I'm running "styleguide server" but I'm not seeing any of controls, just the static pages. Any ideas?

**Comment by miezis** [comment]
@nynban Could you please provide more details? What do you see in devtools console after opening styleguide page?

**Comment by dlewand691** [comment]
I was only able to get it to start using `styleguide start` as well and am not seeing where to enter edit mode. Please clarify.
